src/dom-to-image-more.js

```diff
@@ -1199,13 +1199,31 @@
                 sandbox.style.visibility = 'hidden';
                 sandbox.style.position = 'fixed';
                 document.body.appendChild(sandbox);
-                // Ensure that the iframe is rendered in standard mode
+                sandbox.id = sandbox.contentDocument.title = 'domtoimage-sandbox';
```

Comment:
Worried that we should be using our `uid()` method to synthesize a unique ID (to be used inside here only, obviously) instead of "hoping" it won't clash. This would follow the pattern for the class names we create.

Reply:
Resolved in fd264d9